src/deploy/auto_common/ipmi.py

```diff
@@ -38,7 +39,7 @@ class Ipmi:
             and fix the missing dependencies etc.
         make
         make install
-        Add c:\\cygwin\\bin and c:\\cygwin\\usr\\local\\bin to your path
+        Add r"c:\\cygwin\\bin" and r"c:\\cygwin\\usr\\local\\bin" to your path
         (or the relevant paths for your setup)
         make sure you can talk to one of your servers :
         in cygwin :
```

Comment:
We can actually remove all of the above, this is back from the (very old) days we were supporting installations from windows.

Reply:
I meant the entire comment section can be removed, ipmi ships with rhel the install instructions are no longer relevant.

Reply:
The comment section has been removed from the code.